frontend/src/pages/TermPlanner/ContextMenu/ContextMenu.tsx

```diff
@@ -58,6 +59,11 @@ const ContextMenu = ({ code, plannedFor, ignoreFromProgression }: Props) => {
     ignoreFromProgressionMutation.mutate(code);
   };
 
+  const handleUnilective = () => {
+    window.open(`https://unilectives.devsoc.app/course/${code}`, '_blank');
+    dispatch(addTab(code));
```

Comment:
Im not sure we want to `dispatch(addTab(code))` here. 

This adds the course to the course selector page tabs.

Reply:
ohh right i just saw every url navigation having it so i just also added sorry ;(